src/domain/models/Credential.ts

```diff
@@ -34,7 +35,8 @@ export abstract class Credential implements Pluto.Storable {
 }
 
 export interface ProvableCredential {
-  presentation(): unknown;
+  presentation(): W3CVerifiablePresentation;
+  verifiableCredential(): W3CVerifiableCredential
```

Comment:
is it true that all ProvableCredentials are going to give a W3C VerifiablePresentation?